### ISSUE TITLE
User linked stats

### ISSUE THREAD

**Issue by Jesse-longname** [Jesse-longname/gtc17#16]
Stats should really be linked to the user. To do this, we would have to know how they are matching User Id's to names, but we can always implement this with mocked up data for now.

**Comment by adamsimonini** [comment]
Don't have anything to add at the moment. But yes, we need to collect our thoughts and reply to that E-mail, as we can request resources and information to be made available before the date.

**Comment by Jesse-longname** [comment]
Yeah, I was thinking that too. I sent an email to all of us about the email they sent us.
